framework/db/SchemaBuilderTrait.php

```diff
@@ -51,6 +51,18 @@ public function primaryKey($length = null)
         return $this->getDb()->getSchema()->createColumnSchemaBuilder(Schema::TYPE_PK, $length);
     }
 
+    /**
+     * Creates a small integer primary key column.
+     * @param int $length column size or precision definition.
+     * This parameter will be ignored if not supported by the DBMS.
+     * @return ColumnSchemaBuilder the column instance which can be further customized.
+     * @since 2.16
```

Comment:
2.0.16

Reply:
Thanks! But @samdark added this to the 2.1.1 milestone 21 days ago anyway. I will change.